test/unit/spec/v2/view-builder/views/__snapshots__/AutoRedirectView_spec.js.snap

```diff
@@ -1,5 +1,150 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`v2/view-builder/views/AutoRedirectView Android OV Enrollment Add User Gesture if OV enrollment on Android: should show user gesture 1`] = `
```

Comment:
For this test we want a different title, and to replace the spinner with a button